src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1,4 +1,4 @@
-﻿
```

Comment:
Should this PR add an entry to CHANGES_NEXT_RELEASE file? Has this PR fixes something previously broken or it is only a refactor of things that already were working? (in order to check, just run the new unit test with the old code in master). In the former case, an entry about the fix should be added.

Reply:
Yes, the fix should be added to the file.

Reply:
Fixed in 40da980